test/integration/api/images.bats

```diff
@@ -68,3 +68,15 @@ function teardown() {
 	[ "${#lines[@]}" -eq 2 ]
 	[[ "${lines[1]}" == *"testimage"* ]]
 }
+
+@test "docker images - after build on engine side" {
+	start_docker_with_busybox 2
+	swarm_manage
+
+	docker -H ${HOSTS[0]} build --no-cache -t testimage:latest $TESTDATA/build
+
+	run docker_swarm images testimage
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -eq 2 ]
```

Comment:
question: the `2` here means that `2` images present here? one for `testimage:latest` and one `busybox`?